lesson10/js/main.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+const menuContainer = document.getElementById("js-menu_container");
+const menuLists = document.createElement("ul");
+const menuContents = [
+  {
+    to: "bookmark.html",
+    img: "images/1.png",
+    alt: "画像1",
+    text: "ブックマーク",
+  },
+  {
+    to: "message.html",
+    img: "images/2.png",
+    alt: "画像2",
+    text: "メッセージ",
+  },
+];
+
+const loading = document.getElementById("js-loading");
+
+const loadingImg = document.createElement("img");
+loadingImg.src = "images/loading-circle.gif";
+loadingImg.classList.add("el_loadingImg");
+
+function createMenuList(values) {
+  const fragment = document.createDocumentFragment();
+
+  values.forEach((value) => {
+    const listItem = document.createElement("li");
+
+    const listImg = document.createElement("img");
+    listImg.src = value.img;
+    listImg.alt = value.alt;
+    listImg.width = 40;
+    listImg.height = 40;
+
+    const listAnchor = document.createElement("a");
+    listAnchor.href = value.to;
+    listAnchor.textContent = value.text;
+
+    fragment.appendChild(listItem).appendChild(listAnchor).appendChild(listImg);
+  });
+
+  menuContainer.appendChild(menuLists).appendChild(fragment);
```

Comment:
get  menulist here in this function , because of menuLists refferenced only this function.
no need write with above.
```suggestion
  const menuLists = document.createElement("ul");
  menuContainer.appendChild(menuLists).appendChild(fragment);
```

Reply:
Thank you for review and Suggested change.
Be careful to write the variables used in the function inside.

This  Commit Suggestion.